radix-engine/src/errors.rs

```diff
@@ -150,6 +149,8 @@ pub enum KernelError {
 
     // Invoke
     InvalidInvokeAccess,
+
+    DanglingNodes(Vec<NodeId>),
```

Comment:
I quite like the old name better or `OrphanedNodes`

Reply:
Yes, will change.